src/lib/utils/developerHelper.ts

```diff
@@ -2,7 +2,7 @@ import { useEffect } from 'react';
 import { saveInLocalStorage } from '~root/providers/localStorage.provider';
 
 export const MAXIMUM_VALIDATOR_COUNT_FLAG =  'maximumValidatorCount';
-
+export const TEST_WALLET_ADDRESS =  'testWalletAddress';
```

Comment:
This is a bit too generic, there is no connection between the constants here. Maybe should consider breaking down to different utils files next time we add something else.

Reply:
it's just dev-helper file which contains some dev functions and flags, maybe we will refactor it but for now it's not necessary I think